tests/dotnet/UnitTests/TemplateTest.cs

```diff
@@ -1,28 +1,63 @@
 using System.Text.Json;
-using System.Text.Json.Serialization;
 
 namespace Xamarin.Tests {
+	public enum TemplateLanguage {
+		CSharp,
+		FSharp
```

Comment:
Really minor, but we prefer adding a trailing comma to the end of the last enum value, because if we have to add more values later, the diff will be cleaner:

```suggestion
		FSharp,
```